logs/analyze_test.go

```diff
@@ -4328,6 +4328,9 @@ var tests = []testpair{
 func TestAnalyzeLogLines(t *testing.T) {
 	for _, pair := range tests {
 		l, s := logs.AnalyzeLogLines(pair.logLinesIn)
+		for idx := range l {
+			l[idx].Content = ""
```

Comment:
Multiple tests now do this to avoid needing to copy-paste the log content into the expected output struct. I could do that if desired, it would just be tedious.